### PR TITLE
Add French in the settings languages selector

### PR DESCRIPTION
**Pull request by judemont**
Hello, 
I added French in the language selector of the settings.
I also updated the French translation.

### CODE REVIEW

File: src/setup/locales/fr/translation.json [hunk 1]
```diff
@@ -51,9 +69,12 @@
       "playbackSpeed": "Vitesse"
     },
     "popouts": {
+      "back": "Eevenir en arrière",
```
Comment (BrightDV) [inline src/setup/locales/fr/translation.json:72]:
```suggestion
      "back": "Revenir en arrière",
```

File: src/setup/locales/fr/translation.json [hunk 2]
```diff
@@ -51,9 +69,12 @@
...
       "sources": "Sources",
       "seasons": "Saisons",
       "captions": "Sous-titres",
+      "playbackSpeed": "Vitesse de lecture",
+      "customPlaybackSpeed": "Custom playback speed",
```
Comment (BrightDV) [inline src/setup/locales/fr/translation.json:77]:
```suggestion
      "customPlaybackSpeed": "Vitesse de lecture personnalisée",
```

File: src/setup/locales/fr/translation.json [hunk 3]
```diff
@@ -16,16 +16,34 @@
...
-    "title": "Ce fournisseur a été désactivé",
-    "description": "Nous avons eu des problèmes avec le fournisseur ou bien il était trop instable pour être utilisé, donc nous avons dû le désactiver."
+  "seasons": {
+    "seasonAndEpisode": "S{{saison}} E{{épisode}}"
   },
-  "page": {
-    "title": "Impossible de trouver cette page",
-    "description": "Nous avons cherché partout : sous les poubelles, dans le placard, derrière le proxy, mais nous n'avons finalement pas pu trouver la page que vous recherchez."
+  "notFound": {
+    "genericTitle": "Not found",
```
Comment (BrightDV) [inline src/setup/locales/fr/translation.json:33]:
```suggestion
    "genericTitle": "Introuvable",
```

**Review by BrightDV — COMMENTED** [review]
Only small typos, otherwise, looks great!

**Comment by judemont** [comment]
> Only small typos, otherwise, looks great!

It's done, thanks

File: src/setup/locales/fr/translation.json [hunk 4]
```diff
@@ -51,9 +69,12 @@
       "playbackSpeed": "Vitesse"
     },
     "popouts": {
+      "back": "Revenir en arrière",
```
Comment (JamesHawkinss) [inline src/setup/locales/fr/translation.json:72]: This is quite a long string, suggesting `Retourner` as an alternative

**Review by JamesHawkinss — CHANGES_REQUESTED** [review]

File: src/setup/locales/fr/translation.json (hunk 4 above)
Comment (JamesHawkinss) [inline src/setup/locales/fr/translation.json:72]:
```suggestion
      "back": "Retourner",
```

Comment (JamesHawkinss) [reply src/setup/locales/fr/translation.json:72]: This is quite a long string, suggesting Retourner as an alternative

**Review by JamesHawkinss — APPROVED** [review]

**Pull request merged**